### DISCUSSION TITLE
Profissional FrontEnd entre os 3 mais procurados no LinkedIn

### DISCUSSION THREAD

**Discussion by vicainelli** [frontendbr/forum#1320] [Discussão]
Olá pessoas,

Acabei de ler essa [matéria aqui na Exame](https://exame.abril.com.br/carreira/estes-foram-os-3-profissionais-mais-procurados-em-2018-segundo-o-linkedin/), falando sobre os cargos mais buscados na rede no ano de 2018.

Esse resultado é global, então não sei se isso reflete no nosso mercado brasileiro, o que vocês acham?

**Comment by mateusavila** [comment]
A procura no Brasil tá aquecida, mas isto ocorreu pois muitos devs daqui vão pra fora, e os cargos de nível mais elevado aqui ficam sem pessoas para preencher estas vagas.

**Comment by vitor-mendrone-deel** [comment]
O mercado para desenvolvedores front-end está realmente muito aquecido, é comum eu receber uma quantidade considerável de contatos de recrutadores em um mesmo dia. Isso, em minha humilde opinião, tem dois lados: 

O bom: 
- Melhores salários
- Maior oferta de vagas em empresas legais
- Mais benefícios
- Mais inovação na área

O ruim: 
- Nível técnico dos profissionais cai, as empresas exigem menos qualificação
- Empresas em desespero tentam automatizar ou simplificar o trabalho de front-end gerando aplicações problemáticas e repletas de tech debits.
- Muitos dos profissionais pulam etapas importantes da carreira. Não é raro encontrar dev front-end senior com somente 1 ou 2 anos de carreira.

**Comment by ninetails** [comment]
Não gosto de falar desta maneira, cheio dos _negociês_ mas nem é mais tendência, é atualidade.

Pelo artigo, fica bem claro: empresas querem automatizar e melhorar seus ganhos e entrar na era digital (contratando Engenheiros de Software, que lá cita como maior demanda em setores que não são ligados em tecnologia) e pra TI, são aqueles que querem suprir a demanda desses outros setores para quem prefere não investir por conta em TI.

Nunca saí do país (então posso estar falando besteira) mas o Brasil está bem defasado em relação a tecnologias, faltam APIs pra consumo básico (pra saber o quanto vc gastou de água, luz) e bancos ainda possuem muitos processos manuais. Temos nossa legislação e tributação que dificulta serviços externos a entrarem mas ao mesmo tempo facilita com que novas startups surjam...

E isso tudo reflete sim no nosso mercado. Pessoal bom aqui, vai pra fora. Pra quem ficar, vai sobrar vaga boa, também refletindo na especialização dessa galera que ficou. E por aí vai...
<sup><sub>
A única coisa que falta mesmo é valorizarem mais a gente e a gente se valorizar mais :P
</sub></sup>

**Comment by rubenmarcus** [comment]
Resta saber se esse "aquecimento" não é tipo uma bolha, e se em 2 ou 3 anos , o mercado vai estar esvaziado.

**Comment by caioincau** [comment]
Acho que está bem aquecido e isto é reflexo da migração/criação de novos sistemas usando novas tecnologias como React/Vue/Angular, o pessoal finalmente está percebendo que é mais vantajoso trabalhar com o FE separado do BE e especializando as coisas.

Vale citar que DevOps também está ali e é uma profissão crucial que surgiu "recentemente".

**Comment by deppbrazil** [comment]
## Meu ponto de vista ##

### Sobre a matéria ###
Acho que faltou ressaltar tb os cargos de UX/UI que mesmo em muitas culturas eles são taxados de **Designers da modinha**, tem contribuído muito com esse avanço tecnológico em interfaces e trajetória do usuário dentro do produto.

De fato a demanda por vagas em tecnologia tem crescido muito principalmente para Front nas libs e frameworks hype do mercado (Vue, Angular, React) mas que tb tem sido um problema na minha opnião ao mesmo tempo por 2 motivos ou mais: 
 
* 1º A demanda cresce mais que profissionais habilitados nessas stacks;

* 2º Empresas já querem o cara pronto nível Pleno/Senior pois isso resolve o problema delas, mas poucas pensam em fomentar ambientes que formam esses profissionais, tendo em vista também que graduação não prepara pra isso, e nem deve mas isso faz com que grande porcentagem dos profissionais não tenham oportunidades de aplicar essas stacks pela primeira vez;

### Quanto a "esse aquecimento do mercado acabar" ###
Não aposto que isso vá acontecer tendo em vista que a tendência é sempre ter produtos cada vez mais digitalizados e interfaces cada vez mais diferentes, o que pode ocorrer eu acredito e as estatísticas dizem também são novos profissionais que ainda não existem no mercado como de fato aconteceu de 2013 pra cá... naquela época não se falava de Frontend, Designer UX, Engenheiro Frontend, Frontend UI, whatever... ou se existia não se falava com a frequência que se fala hoje. 

### Resumindo ###
Se tratando de produto web independente da tecnologia A ou B, tudo vai cuspir no final HTML, CSS e Javascript, então a demanda só tem a crescer na minha opinião e o que vai mudar sempre são os meios para vc cuspir isso na tela...

## :) ##
